onadata/libs/utils/model_tools.py

```diff
@@ -1,4 +1,7 @@
-# -*- coding: utf-8 -*-
+# coding: utf-8
+from __future__ import unicode_literals, print_function, division, absolute_import
+# coding: utf-8
+from __future__ import unicode_literals, print_function, division, absolute_import
```

Comment:
oops

Reply:
there's actually a lot more of these than what i've flagged. i was blindly approving `south_migration` files because i believe they're useless now.
```
$ grep __future__ $(ag __future__ | cut -f 1 -d ':' | sort | uniq -c | grep '^ *2 ' | cut -f 2- -d '2')
onadata/apps/api/south_migrations/0002_auto__add_project__add_unique_project_name_organization.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/api/south_migrations/0002_auto__add_project__add_unique_project_name_organization.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/api/south_migrations/0003_auto__add_projectxform__add_unique_projectxform_xform_project.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/api/south_migrations/0003_auto__add_projectxform__add_unique_projectxform_xform_project.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0024_auto__add_field_xform_allows_sms__add_field_xform_sms_id_string.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0024_auto__add_field_xform_allows_sms__add_field_xform_sms_id_string.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0024_auto__chg_field_attachment_mimetype.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0024_auto__chg_field_attachment_mimetype.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0025_auto__add_field_xform_encrypted.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0025_auto__add_field_xform_encrypted.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0025_sms_id_string_filling.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0025_sms_id_string_filling.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0026_auto__chg_field_xform_sms_id_string__add_unique_xform_sms_id_string_us.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0026_auto__chg_field_xform_sms_id_string__add_unique_xform_sms_id_string_us.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0027_added_instance_is_deleted.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0027_added_instance_is_deleted.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0028_update_is_deleted_field.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0028_update_is_deleted_field.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0029_auto__chg_field_attachment_mimetype__add_field_xform_encrypted__add_fi.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0029_auto__chg_field_attachment_mimetype__add_field_xform_encrypted__add_fi.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0030_auto__add_field_xform_num_of_submissions.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0030_auto__add_field_xform_num_of_submissions.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0030_auto__add_ziggyinstance.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0030_auto__add_ziggyinstance.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0031_auto__add_field_xform_last_submission_time.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0031_auto__add_field_xform_last_submission_time.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0031_auto__add_field_ziggyinstance_form_version.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0031_auto__add_field_ziggyinstance_form_version.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0032_added_xform_to_ziggy_submissions.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0032_added_xform_to_ziggy_submissions.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0033_add_model_Note.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0033_add_model_Note.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0033_auto__chg_field_xform_id_string.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0033_auto__chg_field_xform_id_string.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0034_add_field_Instance_json.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0034_add_field_Instance_json.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0035_fill_json.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0035_fill_json.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0036_auto__chg_field_xform_sms_id_string.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0036_auto__chg_field_xform_sms_id_string.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0037_auto__del_field_instance_start_time__del_field_instance_date.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0037_auto__del_field_instance_start_time__del_field_instance_date.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0038_auto__del_field_instance_is_deleted.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0038_auto__del_field_instance_is_deleted.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0039_auto__del_field_xform_surveys_with_geopoints__add_field_xform_instance.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0039_auto__del_field_xform_surveys_with_geopoints__add_field_xform_instance.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0040_auto__add_field_instance_geom.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0040_auto__add_field_instance_geom.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0041_convert_lng_lat_to_points.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0041_convert_lng_lat_to_points.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0042_remove_odk_prefix.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0042_remove_odk_prefix.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0043_xform_title_255.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0043_xform_title_255.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0044_auto__del_field_xform_is_crowd_form.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0044_auto__del_field_xform_is_crowd_form.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0045_auto__add_field_xform_require_auth.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/logger/south_migrations/0045_auto__add_field_xform_require_auth.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0009_auto__add_field_userprofile_address__add_field_userprofile_phonenumber.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0009_auto__add_field_userprofile_address__add_field_userprofile_phonenumber.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0010_auto__add_field_userprofile_created_by.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0010_auto__add_field_userprofile_created_by.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0011_auto__add_field_userprofile_num_of_submissions.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0011_auto__add_field_userprofile_num_of_submissions.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0012_auto__add_unique_metadata_xform_data_type_data_value.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0012_auto__add_unique_metadata_xform_data_type_data_value.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0013_auto__add_field_metadata_file_hash.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0013_auto__add_field_metadata_file_hash.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0014_added_metadata_field_to_user_profile.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/main/south_migrations/0014_added_metadata_field_to_user_profile.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/sms_support/providers/smssync.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/sms_support/providers/smssync.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/sms_support/providers/textit.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/sms_support/providers/textit.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/sms_support/providers/twilio.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/sms_support/providers/twilio.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0014_auto__add_field_export__status.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0014_auto__add_field_export__status.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0015_auto__del_field_export__status__add_field_export_internal_status.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0015_auto__del_field_export__status__add_field_export_internal_status.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0016_auto__add_field_export_export_url.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0016_auto__add_field_export_export_url.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0017_remove_odk_prefix.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/apps/viewer/south_migrations/0017_remove_odk_prefix.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/libs/utils/model_tools.py:from __future__ import unicode_literals, print_function, division, absolute_import
onadata/libs/utils/model_tools.py:from __future__ import unicode_literals, print_function, division, absolute_import
```

Reply:
My batch update script was definitively not good. 
@dorey would have told me: " `git add .` is evil "